frontend/pages/login.vue

```diff
@@ -34,104 +34,132 @@ const canSubmit = computed(() => email.value && password.value && !Object.keys(e
+                  {{ $t('login_and_register.forgotten') }}
+                </BcLink>
+              </div>
+            </label>
+            <InputText
+              id="password"
+              v-model="password"
+              v-bind="passwordAttrs"
+              type="password"
+              :placeholder="$t('login_and_register.password')"
+              :class="{ 'p-invalid': errors?.password }"
+              aria-describedby="text-error"
+            />
+            <div class="p-error">
+              {{ errors?.password }}
+            </div>
           </div>
-          <Button class="button" type="submit" :label="$t('login_and_register.submit_login')" :disabled="!canSubmit" />
-        </div>
-      </form>
+          <div class="last-row">
+            <div class="account-invitation">
+              {{ $t('login_and_register.dont_have_account') }}
+              <BcLink to="/register" :target="Target.Internal" class="link">
+                {{ $t('login_and_register.signup_here') }}
+              </BcLink>
+            </div>
+            <Button class="button" type="submit" :label="$t('login_and_register.submit_login')" :disabled="!canSubmit" />
+          </div>
+        </form>
+      </div>
     </div>
   </BcPageWrapper>
 </template>
 
 <style lang="scss" scoped>
 @use "~/assets/css/fonts.scss";
 
-.container {
-  position: relative;
-  width: 300px;
-  height: 240px;
-  margin: auto;
-  margin-top: 100px;
-  margin-bottom: 50px;
-  padding: var(--padding);
+.page {
+  display: flex;
+  flex-direction: column;
 
-  form {
-    @include fonts.standard_text;
+  .container {
     position: relative;
-    display: flex;
-    flex-direction: column;
-    width: 100%;
-    height: 100%;
-
-    .input-row {
-      position: relative;
-      display: flex;
-      flex-direction: column;
-      margin: var(--padding) auto 0px auto;
-      width: 80%;
+    margin: auto;
+    margin-top: 100px;
+    @media (max-width: 600px) { // mobile
+      margin-top: 0px;
+    }
+    margin-bottom: 30px;
+    padding: var(--padding-large);
+    box-sizing: border-box;
+    width: min(530px, 100%);
 
-      &:first-child {
-        padding-top: 5px;
-      }
-      .label {
-        margin-bottom: 8px;
-      }
+    .title {
+      @include fonts.dialog_header;
+      margin-bottom: var(--padding-large);
     }
 
-    .last-row {
+    form {
+      @include fonts.standard_text;
       position: relative;
       display: flex;
-      margin-top: auto;
+      flex-direction: column;
+      width: 100%;
+      height: 100%;
 
-      .account-invitation {
+      .input-row {
         position: relative;
-        @include fonts.small_text;
-        margin-right: auto;
+        display: flex;
+        flex-direction: column;
+        height: 100px;
+        .label {
+          display: flex;
+          flex-direction: row;
+          margin-bottom: 12px;
+          .right-cell {
```

Comment:
According to Figma the forgot password link should have the small font size, like the register link

Reply:
OK, done :)